esphome/components/voice_assistant/voice_assistant.cpp

```diff
@@ -435,6 +435,10 @@ void VoiceAssistant::loop() {
         this->set_timeout("playing", 50, [this]() {
           this->cancel_timeout("speaker-timeout");
           this->set_state_(State::IDLE, State::IDLE);
+
+          api::VoiceAssistantAnnounceFinished msg;
+          msg.success = true;
+          this->api_client_->send_voice_assistant_announce_finished(msg);
```

Comment:
Does this really need to send on every tts?

Reply:
Yes, we need to know when the device is done speaking to match the state in HA.